pkg/fusionauth/Client.go

```diff
@@ -7008,6 +7008,36 @@ func (c *FusionAuthClient) RetrieveUserByLoginIdWithContext(ctx context.Context,
 	return &resp, &errors, err
 }
 
+// RetrieveUserByLoginIdWithLoginIdTypes
+// Retrieves the user for the loginId, using specific loginIdTypes.
+//
+//	string loginId The email or username of the user.
+//	[]string loginIdTypes the identity types that FusionAuth will compare the loginId to. Defaults to [email, username]
```

Comment:
same comment as java about optional, but not sure how much we want to care about that